src/test/scala/beam/router/TollRoutingSpec.scala

```diff
@@ -100,16 +106,13 @@ class TollRoutingSpec
             Modes.BeamMode.CAR,
             asDriver = true
           )
-        ),
-        Access,
-        false,
-        timeValueOfMoney
+        )
```

Comment:
@zishanbilal Errr, I guess this was an oversight, but this test was actually testing something. Could you please revert those assertions and restore that functionality?

Reply:
(Kind of urgent, since we would like to merge a cost-related PR..)